### ISSUE TITLE
Can RSSPreview display <lastBuildDate>?

### ISSUE THREAD

**Issue by johnhawkinson** [aureliendavid/rsspreview#57]
Hi!

I really appreciate RSSPreview and it's done a stellar job replicating the functionality that Firefox removed.  However, I frequently find myself wanting to check the date of an RSS feed (to understand how up-to-date it is for infrequently-generated feeds) , and I have to Tools > Web Developer > Page Source to get there, wait for it to (re-download?) and render, which is cumbersome, all to inspect the one XML tag at the top:

```xml
<lastBuildDate>Wed, 15 Jul 2020 15:00:01 GMT</lastBuildDate>
```

Is it feasible for RSSPreview to display this information, just as it displays the `<title>` and `<description>` tags?

An example such RSS feed: https://ecf.nysd.uscourts.gov/cgi-bin/rss_outside.pl

I'm happy to work on a pull request if it would be accepted.

Thanks!

**Comment by johnhawkinson** [comment]
Wow, you are amazingly fast, thank you!
I agree with @jonafato that it would be a bit better if it said "Last build date:" or similar in front.

I would have initially put the lastBuildDate after the Description field, but I think your choice to put it beforehand is a better one!

**Comment by aureliendavid** [comment]
yeah I thought about adding a label before the date but I wanted to avoid the whole i18n issue so I took the lazy road

I guess I'll have to do it at some point since the options are still english-only

I'll push a version with the label on AMO soon and think about i18n for later

**Comment by aureliendavid** [comment]
this is now included in v3.14

you can close the issue if it's all ok

thanks

**Comment by johnhawkinson** [comment]
👍 Looks great.

<img width="1081" alt="Screen Shot 2020-07-15 at 16 19 02" src="https://user-images.githubusercontent.com/1270718/87591868-fed5e680-c6b6-11ea-920b-d922d41486c8.png">
